rts/Rendering/Models/3DModel.cpp

```diff
@@ -246,11 +246,7 @@ void S3DModelPiece::Shatter(float pieceChance, int modelType, int texType, int t
 
 void S3DModelPiece::SetPieceTransform(const Transform& parentTra)
 {
-	bposeTransform = parentTra * Transform{
-		CQuaternion(),
-		offset,
-		scale
-	};
+	bposeTransform = parentTra * ComposeTransform(offset, ZeroVector, scale);
```

Comment:
N2S, more in line with the previous implementation https://github.com/beyond-all-reason/RecoilEngine/blob/rel2503/rts/Rendering/Models/3DModel.h#L202